pkgs/development/interpreters/python/pypy/2.7/default.nix

```diff
@@ -3,13 +3,13 @@
 , makeWrapper, callPackage, self, gdbm, db
 , python-setup-hook
 # For the Python package set
-, pkgs, packageOverrides ? (self: super: {})
+, pkgs, pythonPackages, packageOverrides ? (self: super: {})
```

Comment:
The packages set is an attribute of the interpreter, `python`. Please use that instead.